src/dfilemaker/dfilemaker.c

```diff
@@ -566,6 +616,29 @@ int main(int narg, char** arg)
     int* lind; // list of ints in order to resort things
     int initsum, noff;
 
+    int kft=0,ifac=0; // kft is index of filltype
+    unsigned long long sizeminl,sizemaxl;  // for mfu_abtoul
+    long int sizemin=0,sizemax=0;
```

Comment:
For file sizes, let's store these as uint64_t to be consistent with the other tools.

Reply:
Addressed in #604